addons/account_payment/controllers/portal.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo.addons.account.controllers.portal import PortalAccount
+from odoo.addons.account.controllers.portal import PortalAccount as PA
 from odoo.http import request
 
 
-class PortalAccount(PortalAccount):
+class PortalAccount(PA):
```

Comment:
Why not real names likes AccountPortalAccount (well this one sounds weird, but you got the idea :D)

Reply:
I tried that approach originally but then realized something: I'd have to also change all imports and inheritances in downstream modules and potentially break third-party apps too :scream:

E.g., if module `account_foo` imports `account_payment`'s `PortalAccount`, I'll have to rename it to `AccountPortalAccount` in `account_payment` and in `account_foo` too

I can change it depending on what the general consensus is, I just did it the lazy way for now :stuck_out_tongue_winking_eye:

Reply:
Indeed. Minimal patch.

And the reverse move ? Import PortalAccount as BasePortalAccount ?

Reply:
Also works, I did that for some of them that were only one word (e.g. `Home` -> `BaseHome`) because it felt weird otherwise to just import it as a single uppercase later.

I will wait for some more input and based on that we'll see which option is preferred by everyone